src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -17,13 +22,49 @@ private Drivetrain() {
      * @param leftSpeed  Speed of left side.
      * @param rightSpeed Speed of right side.
      */
-    public void tankDrive(double leftSpeed, double rightSpeed) {
+    public void tankDrive(double leftPower, double rightPower)
+    {
```

Comment:
Formatting, formatting, formatting. `{` goes on the same line as the method declaration.